### ISSUE TITLE
Will not run in node 4.1.0 installed first

### ISSUE THREAD

**Issue by SteveALee** [nanjingboy/nvmw#8]
- install node 4.1.0
- install nvmw
- run nvmw ls

C:\Users\Steve\AppData\Roaming\npm\node_modules\nvmw\bin\nvmw.b
at"
ERROR: The system was unable to find the specified registry key or value.

It seems that the InstallPath key is not set

**Comment by narendranss** [comment]
I too encountered the same issue.

**Comment by meeximum** [comment]
me too

**Comment by SteveALee** [comment]
@narendranss @meeximum I searched for a solution and found [nvm-windows](https://github.com/coreybutler/nvm-windows) does work. In fact it simply manages a symbolic link so I could hack using mklink from the command line until they added 4.x.x support. Might be a useful temp solution for you too.

**Comment by keerthivasan-r** [comment]
Still, it happens :worried:

**Comment by vmourac** [comment]
Any updates on this? I have the same issue =(

**Comment by abhijeet1403** [comment]
Still happening in 6.3.0 . Am i doing something wrong or support is over?

**Comment by wayoung** [comment]
+1

**Comment by aluanhaddad** [comment]
Had the same problem just today. I've never used this package before.

I looked at `nvmw.bat` and found it was querying the registry for a value named `InstallPath` under the key `HKEY_CURRENT_USER\Software\Node.js`.

I didn't have such a value under that key.

I created a value named `InstallPath` of type `REG_SZ` pointing to my local installation directory. `C:\Program Files\nodejs`

Unfortunately it fails without a stack trace or any other information when I try to install any recent version of Node, but that is a separate issue.
